GOALS.md

```diff
@@ -0,0 +1,55 @@
+
+There should be an addendum which covers how various safety standards like ISO 26262 map onto the coding guidelines.
+
+This serves as a tracking issue which is why it's considered "XL". Work will get logged against this in smaller chunks!
+
+# Way of work
+
+## Outline & issue breakdown
+
+We will use the Coding Guidelines Work Items [board](https://github.com/orgs/rustfoundation/projects/1) as a means to break the work down into smaller chunks that can be tackled in a reasonable manner.
+
+## Contribution of existing guidelines
+
+We are very open to receiving contributed coding guidelines in whole or in part and wholly originally contributions based on learnings from past organizational experience using Rust in safety-critical projects.
+
+## Contribution of a new guideline
+
+A good first step is to open a new [coding guideline issue](https://github.com/rustfoundation/safety-critical-rust-coding-guidelines/issues/new?template=CODING-GUIDELINE.yml).
+
+# Goals
+
+* Coding guidelines that make a "best effort" attempt at cataloging common pieces (e.g. struct or unsafe) of the Rust programming language and how they fit into a safety-critical project
+  * We will use [MISRA Compliance: 2020](https://misra.org.uk/app/uploads/2021/06/MISRA-Compliance-2020.pdf) for categorization
+  * Includes rationale with links to parts of the Rust Project and wider Rust community for guidance
+  * Could later be refined according to various standards, e.g. DO 178 or ISO 26262
+* Practical recommendations on how to use this piece of the language
+  * May include considerations of "what" is being built, e.g. broadly speaking library software: (potentially broke down further into low-level driver code, a framework system for real-time applications, SDKs) vs application software
+* Should be done in parallel with developing an addendum matrix to reduce burden of attaching these later
+  * We can begin with DO 178 and ISO 26262 at perhaps chapter level, maybe subsection level _for now_ and expand later
+* Releases of the coding guidelines are tagged and released with versions of the stable compiler which they support
+* Upstream Clippy lints which will cover decidable guidelines
+
+## Goals obtained by discussion with Tooling Subcommittee
+
+* Make a label for each which _in theory_ is decidable or not
+* Include for each a minimum of one compliant and non-compliant example of code to illustrate point
+* Consider only the language reference / spec, not the tooling availability when writing the coding guidelines
+* Evidence-based with statistics around human error when programming Rust to support both 1. what guidelines are written and 2. why a suggestion was made
```

Comment:
This is perfect. I'm just writing this so I remember, or to ask you to remind me, to open an Issue to start talking about how to gather this information. I have a couple of ideas, but I really have to open that issue. I want to reference this document in the issue; hence why I want to open it afterwards and not right now as I'm writing this :)

Reply:
Well, perhaps I'd change it somewhat to this:

> * Guidelines should be evidence-based, with statistics around human error when programming Rust, to support:
>   1. What guidelines are written, and 
>   2. Why a specific suggestion was made